cfme/tests/webui/test_expression_editor.py

```diff
@@ -0,0 +1,43 @@
+import fauxfactory
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+
+class TestDeleteExpressions(object):
+    filter_value = 'beautifulpotato'
+    filt = "Cloud Volume : Name"
+    collection = "volumes"  # any collection with advanced search
+    destination = "All"
+
+    @pytest.mark.meta(automates=[BZ(1720216)])
+    @pytest.mark.parametrize("operator", ["not", "and", "or"])
+    @pytest.mark.uncollectif(lambda operator, appliance:
```

Comment:
Please provide reason of `uncollect`. It will be useful to understand test why `not` operator not applicable for `5.10`

Reply:
good idea, thanks